.github/actions/run-tests-in-container/Dockerfile

```diff
@@ -9,12 +9,20 @@
 # Base Image
 FROM alpine
 
+
 # Testing Dependencies
+# if wanting to install cpp-coveralls via pip install --user cpp-coveralls, these deps are required:
+# python3-dev py3-pip libffi-dev rust cargo openssl-dev git 
```

Comment:
This is worth recording into the commit history (in case we change our minds later), but I think it may cause confusion if someone new were trying to wrap their head around the CI.

Let's leave it on this commit, but remove it in a subsequent commit.

Reply:
With the latest changes, this comment block no longer makes sense, because it would be part of the `publish_coverage` Dockerfile steps instead.

Reply:
I thought I'd removed it. Will do that.